style.css

```diff
@@ -0,0 +1,516 @@
+  }
+
+  .content__pic {
+    height: 22vh;
+  }
+
+  .content__info {
+    max-width: 30vw;
+  }
+
+  .content__text {
+    width: 33vw;
+    font-size: 1em;
+    padding-left: 0.8em;
+    margin: 1.9em 0 1.4em 0;
+    letter-spacing: 0.005em;
+  }
+
+  .content__button {
+    font-size: 1.1em;
+    margin-left: 0.9em;
+  }
+
+  .gallery__item {
+    padding-right: 3.8%;
+  }
+}
+
+@media (min-width:1025px) and (max-width: 1366px) {
+  .wrap {
+    max-width: 1366px;
+  }
+
+  .header__logowrap {
+    padding-left: 5.3%;
+  }
+
+  .header__logo {
+    max-width: 19%;
+  }
+
+  .menu__link {
+    font-size: 16px;
+    margin-right: 26px;
+  }
+
+  .menu__link:last-child {
+    margin-right: 0;
+  }
+
+  .content {
+    height: 66.7vh;
+  }
+
+  .content__info {
+    max-width: 30vw;
+  }
+
+  .content__pic {
+    height: 24vh;
+  }
+
+  .content__text {
+    font-size: 0.8em;
+    padding-left: 0.5em;
+    margin: 2em 0 1.4em 0;
+    letter-spacing: 0.004em;
+  }
+
+  .content__button {
+    font-size: 0.8em;
+    margin-left: 0.85em;
+  }
+
+  .gallery__item {
+    padding-right: 3.85%;
+  }
+}
+
+@media (min-width:768px) and (max-width:1024px) {
+  .wrap {
+    max-width: 1024px;
+  }
+
+  .header {
+    margin: 0 auto;
+    max-width: 92%;
+    height: 22vh;
+  }
+
+  .header__logo {
+    max-width: 22%;
+  }
+
+  .header__logowrap {
+    padding-left: 0;
+  }
+
+  .header__menu {
+    margin-right: .65%;
+  }
+
+  .menu__link {
+    font-size: 0.8em;
+    margin-right: 5.3%;
+  }
+
+
+  .content {
+    height: 67vh;
+    justify-content: flex-start;
+  }
+
+  .content__info {
+    max-width: 37vw;
+    padding-left: 5vw;
+  }
+
+  .content__pic {
+    height: 22.9vh;
+    background-size: 95%;
+  }
+
+  .content__indo {
+    max-width: 85%;
+  }
+
+  .content__text {
+    width: 104%;
+    font-size: 16.1px;
+    margin: 35px 0 24px 0;
+    padding-left: 8.7px;
+  }
+
+  .content__button {
+    font-size: 0.8em;
+    margin-left: 10px;
+    width: 19.7vw;
+  }
+
+  .gallery {
+    height: 68vh;
+  }
+
+  .gallery__wrap {
+    width: 100%;
+  }
+
+  .gallery__first {
+    max-width: 44.08%;
+    padding-right: 4.69%
+  }
+
+  .gallery__second {
+    max-width: 39.51%;
+    padding-right: 4.69%;
+  }
+
+  .gallery__third {
+    max-width: 7.03%;
+    overflow: hidden;
+  }
+
+  .img3 {
+    padding-right: 0;
+    max-width: 407%;
+  }
+}
+
+@media (max-width: 767px) {
+  .wrap {
+    max-width: 767px;
+    background-position: 54%;
+    height: 118vh;
+    margin: 0 auto;
+  }
+
+  .header {
+    height: 18.4vh;
+  }
+
+  .header__logowrap {
+    padding-left: 6%;
+  }
+
+
+  .header__logo {
+    width: 38%;
+    padding-left: 0;
+  }
+
+  .menu__link {
+    display: none;
+  }
+
+  .burger {
+    display: block;
+    position: relative;
+    width: 36px;
+    height: 17px;
+    margin-top: 3%;
+  }
+
+  .burger__bar {
+    display: block;
+    position: absolute;
+    width: 25px;
+    height: 3px;
+    background: #FCF5ED;
```

Comment:
Codacy found an issue: [Expected "#FCF5ED" to be "#fcf5ed" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/VeronikasBaliProject/pullRequest?prid=5951832)